src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -834,89 +819,111 @@
-									VerticalAlignment="Top">
+							<!-- 0: Icon -->
+							<Border Grid.Column="0"
+									Width="40"
+									Height="40"
+									VerticalAlignment="Top"
+									Visibility="{Binding Path=(ut:ControlExtensions.Icon), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource NullToCollapsedConverter}, FallbackValue=Collapsed, TargetNullValue=Collapsed}">
 								<ContentPresenter x:Name="IconPresenter"
 												  Content="{Binding Path=(ut:ControlExtensions.Icon), RelativeSource={RelativeSource TemplatedParent}}"
-												  HorizontalAlignment="Center"
-												  Width="20"
-												  Margin="1,2,18,0"
 												  Foreground="{ThemeResource TextBoxLeadingIconForeground}"
-												  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-												  Visibility="{Binding Path=(ut:ControlExtensions.Icon), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource NullToCollapsedConverter}, FallbackValue=Collapsed, TargetNullValue=Collapsed}" />
+												  Width="20"
+												  Height="20"
+												  HorizontalAlignment="Center"
+												  VerticalAlignment="Center" />
 							</Border>
 
-							<ScrollViewer x:Name="ContentElement"
-										  Grid.Column="1"
-										  HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
-										  HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
-										  IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
-										  IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
-										  IsTabStop="False"
-										  IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
-										  VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
-										  VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
-										  VerticalAlignment="Top"
-										  Padding="0,10"
-										  ZoomMode="Disabled"
-										  AutomationProperties.AccessibilityView="Raw">
-								<ScrollViewer.RenderTransform>
-									<CompositeTransform x:Name="ContentElement_CompositeTransform" />
-								</ScrollViewer.RenderTransform>
-							</ScrollViewer>
+							<!-- 1: Text Elements Container -->
+							<Grid Grid.Column="1" Margin="8,0">
+								<!--
+									We need to center this in both Singleline and Multiline mode:
+									- SingleLine via Border.MinHeight + SV.HAlign
+									- Multiline via Border.Padding
+									The reason we are using an uneven Border.Padding is because the actual platform-specific text input element
+									have different height on each platform (win=21, skia=20). The 10-top guarantee that the 1st line
+									in multiline mode remains the same as in singleline. The 8-bottom leaves room for growth if needed,
+									so we don't grow pass the MaterialOutlinedTextBoxMinHeight of 56.
```

Comment:
```suggestion
									We need to center this in both Singleline and Multiline mode:
									- SingleLine via Border.MinHeight + SV.HAlign
									- Multiline via Border.Padding
									The reason we are using an uneven Border.Padding is because the actual platform-specific text input element
									have different heights on each platform (win=21, skia=20). The 10-top guarantee that the 1st line
									in multiline mode remains the same as in singleline. The 8-bottom leaves room for growth if needed,
									so we don't grow past the MaterialOutlinedTextBoxMinHeight of 56.
```